Source/WebCore/Modules/permissions/NavigatorPermissions.cpp

```diff
@@ -25,7 +25,6 @@
 
 #include "config.h"
 #include "NavigatorPermissions.h"
-
```

Comment:
This is incorrect, please keep the blank line. This is WebKit style.

Reply:
Fixed!